docs/en/cloud/security/gcp-private-service-connect.md

```diff
@@ -0,0 +1,224 @@
+- **Target service**: Please use the entry from the **Service Attachment** column from the **Supported regions** table.
+- **Endpoint name**: Please set a name for the PSC **Endpoint name**.
+- **Network/Subnetwork/IP address**: Please choose the network you want to use for the connection.  You will need to create an IP address or use an existing one for the Private Service Connect endpoint.
+- To make the endpoint available from any region, you can enable the **Enable global access** checkbox.
+
+![Enable Global Access](@site/docs/en/cloud/security/images/gcp-psc-enable-global-access.png)
+
+
+To create the PSC Endpoint, use the **ADD ENDPOINT** button.
+
+The **Status** column will change from **Pending** to **Accepted** once the connection is approved.
+
+![Accepted](@site/docs/en/cloud/security/images/gcp-psc-copy-connection-id.png)
+
+Please copy **PSC Connection ID** & **IP address**(10.142.0.2 in this example), you will
+need this information in next steps.
+
+
+## Setting up DNS
+
+Two options are presented, using the Google Cloud console, and using the gcloud CLI.
+
+### Using the Google Cloud console
+
+- Please create Private DNS zone from **Supported regions** part:
+- Please open **Network services -> Cloud DNS**.
+- Click **Create Zone**:
+
+![Create Zone](@site/docs/en/cloud/security/images/gcp-psc-create-zone.png)
+
+In the Zone Type dialog Please set:
+- Zone type: **Private**
+- Zone name: please set zone name.
+- DNS name: please use the **Private DNS domain** column from the **Supported regions** table for your region.
+- Networks: please attach a DNS zone to networks you are planning to use for connections to ClickHouse Cloud via PSC
+
+![Zone Type](@site/docs/en/cloud/security/images/gcp-psc-zone-type.png)
+
+#### Create a wildcard record
+
+Please point it to the IP address created in the **Adding PSC Connection** step.
+
+![Wildcard DNS](@site/docs/en/cloud/security/images/gcp-psc-wildcard-dns.png)
+
+
+### Using the gcloud CLI
+
+
+#### Create DNS zone
+
+```bash
+gcloud dns \
+  --project=_PROJECTID_ \
+  managed-zones create ch-cloud-us-central1 \
+  --description="Private DNS zone for PSC" \
+  --dns-name="us-central1.p.gcp.clickhouse.cloud." \
+  --visibility="private" \
+  --networks="https://www.googleapis.com/compute/v1/projects/_PROJECTID_/global/networks/default"
+```
+
+#### Create wildcard DNS record
+
+```bash
+gcloud dns \
+  --project=_PROJECTID_ \
+  record-sets create *.us-central1.p.gcp.clickhouse.cloud. \
+  --zone="ch-cloud-us-central1" \
+  --type="A" \
+  --ttl="300" \
+  --rrdatas="10.128.0.2"
+```
+
+## Verify DNS setup
+
+Any record within the us-central1.p.gcp.clickhouse.cloud domain should be pointed
+to Private Service Connect Endpoint IP. (10.142.0.2 in this example). 
+
+```bash
+ping instance-id.us-central1.p.gcp.clickhouse.cloud.
+```
+```response
+PING instance-id.us-east1.p.gcp.clickhouse.cloud (10.142.0.2) 56(84) bytes of data.
+```
+
+## Provide information to ClickHouse support
```

Comment:
@melvynator can you verify this?  What is a "ClickHouse Cloud service ID"? Is that the URL from the connection string?

Reply:
Yeah user can provide us the connection url. Let's follow AWS doc for this: https://clickhouse-docs-ovmhzlgn3-clickhouse.vercel.app/docs/en/manage/security/aws-privatelink#reach-out-to-clickhouse-support